src/components/Deck.scss

```diff
@@ -1,111 +0,0 @@
-$card-width: 40vh;
-
-* {
-    box-sizing: border-box;
-}
-
-html,
-body {
-    margin: 0;
```

Comment:
Global styles are missing, should be added using solution appropriate for `styled-components`

Reply:
Fixed with https://github.com/Greenheart/swipeforfuture.com/pull/125/commits/1e78810b41676815f42ce34ae31a58a9ab4aaef6